openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+start_service() {
```

Comment:
Please consider moving all configuration actions into a `config_openwisp_monitoring` function, and startup into a `start_service` function.(https://github.com/openwrt/packages/blob/791892d482c2255c8681b51bb0b78aa06d48f0e0/utils/watchcat/files/watchcat.init)